shoop/core/models/_orders.py

```diff
@@ -347,6 +347,8 @@ def save(self, *args, **kwargs):
         super(Order, self).save(*args, **kwargs)
         if first_save:  # Have to do a double save the first time around to be able to save identifiers
             self._save_identifiers()
+        for line in self.lines.exclude(product_id=None):
+            line.supplier.module.update_stock(line.product_id)
```

Comment:
I am pretty sure there is cases when order changes (like cancelled) and order lines is not touched and it has effects on stock values. And line can have only one supplier and supplier can have only one module.